README.md

```diff
@@ -1,2 +1,2 @@
-inloco/actions-upload-assets
```

Comment:
Upload become download? 🤔

Reply:
Initial commit in master was copied from the other action.